### PR TITLE
create samarth8765.json

### PR DESCRIPTION
**Pull request by samarth8765**

### CODE REVIEW

**Review by Pradumnasaraf — COMMENTED** [review]
Hey, 

Add your profile to the `data` folder. which is on root of the project

**Comment by samarth8765** [comment]
Hey, @Pradumnasaraf thanks for helping.
I have added my profile to the data folder

**Review by Sindhuinti — DISMISSED** [review]
Looks good 👍🏻

File: data/samarth8765.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  "avatar": "https://github.com/samarth8765.png",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/samd_twts",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "LinkedIn",
+      "url": "https://twitter.com/samarthdhawan2911",
```
Comment (kumarsonsoff3) [inline data/samarth8765.json:15]: Please update it with your LinkedIn URL! 🛠

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

**Comment by samarth8765** [comment]
@kumarsonsoff3 Thanks for the update....I have updated the linkedIn url

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Pull request merged**